src/app/components/page-aside/PageAside.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './PageAside.css';
+
+export function PageAside({
+  bottomMargin = 0,
+  children,
+  className
+}: {
+  children: React.ReactNode;
+  bottomMargin?: number;
+  className?: string;
+}) {
+  return (
+    <div className="page-aside__container">
+      <aside className={className}>{children}</aside>
+      {bottomMargin ? (
+        <div style={{ height: `${bottomMargin}rem`, display: 'block' }} />
```

Comment:
due to the absolutely position nature of the aside and the fixed height requirement on the discord iframe that lives within an aside instance it becomes difficult to add additional space below the aside to ensure global site vertical scrolling allows the entire aside to be visible.

undesirable current behavior: 
unable to scroll to show cutoff aside content
![msedge_fQiRSqNtX6](https://github.com/user-attachments/assets/846fac73-92dc-4a72-97c6-1a888d58c735)

after this CSS merges:
![msedge_Ex2k38Ev7Y](https://github.com/user-attachments/assets/b075e5fb-7ec3-4979-ab87-3f04d450c947)